test/app.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from './test-app/app.module';
+import * as Logger from 'bunyan';
+
+describe('AppController (e2e)', () => {
+  let app: INestApplication;
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+    app = moduleFixture.createNestApplication();
+    await app.init();
+  });
+
```

Comment:
Usually there will be a `afterAll` which will call the `await app.close()` method to close the app.

Reply:
sure, adding